### PR TITLE
Fixed #34

### LINKED ISSUE #34: Addition of function to return the adjacent connected nodes for a given node

### PR DESCRIPTION
**Pull request by Blake2912**
- Added GET route to fetch all nodes
 - Added POST route to fetch all possible neighbours and handled edge cases

### CODE REVIEW

**Comment by Blake2912** [comment]
In order to test the APIs use postman,
 - `/fetch-all-nodes` simple GET request should do the tick and the expected result will be:
 ```
[ "cmrit_entrance", "teacher_parking", "volley_ball_court", "basic_science", "ganesha_statue", "hostel_turn" ]
 ```
- `/fetch-neighbours` simple POST request the payload should have the key as `current_node` and the value should be any valid node returned from the GET call, and the expected response for `teacher_parking`
```
[ "volley_ball_court", "cmrit_entrance", "basic_science" ]
```
Note: It should be a form data for the POST call

File: app.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 from geocodeTree import GeocodeTree
 import test_coordinates as tc
+from cmritPath import CmritField
 
 app = Flask(__name__)
 
 driver = Drive()
 gt = GeocodeTree()
-
+master_graph = CmritField() # This
```
Comment (amartyanambiar) [inline app.py:13]: This is a separate Graph object initialised for Issue #34

**Review by amartyanambiar — APPROVED** [review]
